xbmc/cores/RetroPlayer/process/RPProcessInfo.cpp

```diff
@@ -88,7 +88,7 @@ std::unique_ptr<CRPProcessInfo> CRPProcessInfo::CreateInstance()
   return processInfo;
 }
 
-void CRPProcessInfo::RegisterProcessControl(CreateRPProcessControl createFunc)
+void CRPProcessInfo::RegisterProcessControl(const CreateRPProcessControl& createFunc)
```

Comment:
@lrusak I just came across https://github.com/xbmc/xbmc/pull/22185 again and wondered what you think about this change.